src/api/Controllers/UploadController.cs

```diff
@@ -147,6 +153,54 @@ public async Task<ActionResult<int>> UploadFileAsync(int workgroupId, IFormFile
                 }
             }
 
+            // Add lithology imports if provided
+            if (lithologyImports != null)
+            {
+                // Get the kind id of a lithostratigraphy.
+                var lithoStratiKindId = context.Codelists.FirstOrDefault(cl => cl.Schema == "layer_kind" && cl.IsDefault == true).Id;
```

Comment:
PP: Da es nur einen geben kann, wäre `Single()` besser geeignet. Zudem kann man `cl.IsDefault == true` vereinfachen:
```suggestion
                var lithoStratiKindId = context.Codelists.Single(cl => cl.Schema == "layer_kind" && cl.IsDefault.GetValueOrDefault()).Id;
```

Reply:
Habe ich versucht aber .GetValueOrDefault() kann nicht übersetzt werden.